WorkloadManagementSystem/Service/JobMonitoringHandler.py

```diff
@@ -385,10 +385,10 @@ def export_getJobPageSummaryWeb(self, selectDict, sortList, startItem, maxItems,
       orderAttribute = None
 
     statusDict = {}
-    result = gJobDB.getCounters('Jobs', ['Status'], selectDict,
-                                newer=startDate,
-                                older=endDate,
-                                timeStamp='LastUpdateTime')
+    result = self.gJobDB.getCounters('Jobs', ['Status'], selectDict,
```

Comment:
`cls.gJobDB`

Reply:
no it's not a class method. Same for the others